layers/+lang/emacs-lisp/packages.el

```diff
@@ -61,10 +71,17 @@
           "mhh" 'elisp-slime-nav-describe-elisp-thing-at-point)))))
 
 (defun emacs-lisp/init-emacs-lisp ()
+  (add-hook 'emacs-lisp-mode-hook
+            (lambda () (auto-compile-on-save-mode 1)))
+  (add-hook 'lisp-interaction-mode-hook
+            (lambda () (auto-compile-on-save-mode 1)))
```

Comment:
I think that these two lines actually belong to `init-auto-compile`, don't they?  After all, disabling the `auto-compile` package in `.spacemacs`—which is what I'd do probably—should really disable auto-compilation, shouldn't it?